### PR TITLE
Vidani/mmeft example notebooks

### PR DESCRIPTION
**Pull request by vivek-dani**
# Description
Example notebooks for finetuning MMEFT model

# Checklist

- [x] I have read the [contribution guidelines](https://github.com/Azure/azureml-examples/blob/main/CONTRIBUTING.md)
- [x] I have coordinated with the docs team (mldocs@microsoft.com) if this PR deletes files or changes any file names or file extensions.
- [x] Pull request includes test coverage for the included changes.
- [x] This notebook or file is added to the [CODEOWNERS](https://github.com/Azure/azureml-examples/blob/main/.github/CODEOWNERS) file, pointing to the author or the author's team.

### CODE REVIEW

**Comment by shubhamiit** [comment]
optimizer is repeated twice in component args, and value is also 0.001, please check.

**Comment by shubhamiit** [comment]
SDK- https://ml.azure.com/experiments/id/d0bc365b-16d9-4924-a02e-17ac3376c391/runs/e713cb0a-39ca-4ad2-9af8-f7f16fa771ed?wsid=/subscriptions/dbd697c3-ef40-488f-83e6-5ad4dfb78f9b/resourcegroups/vidani_rg/workspaces/vidani_wrk&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#

batch endpoint - https://ml.azure.com/endpoints/batch/multimodal-classif-1699349198/detail?wsid=/subscriptions/dbd697c3-ef40-488f-83e6-5ad4dfb78f9b/resourcegroups/vidani_rg/workspaces/vidani_wrk&tid=72f988bf-86f1-41af-91ab-2d7cd011db47

File: sdk/python/foundation-models/system/inference/multimodal-classification/multimodal-classification-batch-endpoint.ipynb [hunk 1]
```diff
@@ -0,0 +1,479 @@
...
+    "print(df.head())"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 4. Deploy the model to a batch endpoint\n",
+    "Batch endpoints are endpoints that are used to do batch inferencing on large volumes of data over a period of time. The endpoints receive pointers to data and run jobs asynchronously to process the data in parallel on compute clusters. Batch endpoints store outputs to a data store for further analysis. For more information on batch endpoints and deployments see [What are batch endpoints?](https://learn.microsoft.com/en-us/azure/machine-learning/concept-endpoints?view=azureml-api-2#what-are-batch-endpoints).\n",
```
Comment (rjaincc) [inline sdk/python/foundation-models/system/inference/multimodal-classification/multimodal-classification-batch-endpoint.ipynb:234]:
```suggestion
    "Batch endpoints are endpoints that are used to do batch inferencing on large volumes of data over a period of time. The endpoints receive pointers to data and run jobs asynchronously to process the data in parallel on compute clusters. Batch endpoints store outputs to a data store for further analysis. For more information on batch endpoints and deployments see [What are batch endpoints?](https://learn.microsoft.com/en-us/azure/machine-learning/concept-endpoints?view=azureml-api-2#what-are-batch-endpoints)\n",
```

File: sdk/python/foundation-models/system/inference/multimodal-classification/multimodal-classification-batch-endpoint.ipynb [hunk 2]
```diff
@@ -0,0 +1,479 @@
...
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 3. Prepare data for inference\n",
+    "\n",
+    "We will use the [AirBnb](https://cvbp-secondary.z19.web.core.windows.net/datasets/multimodal_classification/AirBnb.zip) dataset for multi-class classification task. It has a `.csv` file with features and label. Along with it, images are stored separately in `room_images` folder. Column name that stores label is `room_type`.\n",
```
Comment (rjaincc) [inline sdk/python/foundation-models/system/inference/multimodal-classification/multimodal-classification-batch-endpoint.ipynb:158]:
```suggestion
    "We will use the [AirBnb](https://cvbp-secondary.z19.web.core.windows.net/datasets/multimodal_classification/AirBnb.zip) dataset for multi-class or single-label classification task. It has a `.csv` file with features and label. Along with it, images are stored separately in `room_images` folder. Column name that stores label is `room_type`.\n",
```

File: sdk/python/foundation-models/system/inference/multimodal-classification/multimodal-classification-batch-endpoint.ipynb [hunk 3]
```diff
@@ -0,0 +1,479 @@
...
+   "source": [
+    "from azure.ai.ml.entities import (\n",
+    "    BatchEndpoint,\n",
+    "    BatchDeployment,\n",
+    "    BatchRetrySettings,\n",
+    "    AmlCompute,\n",
+    ")\n",
+    "\n",
+    "# Endpoint names need to be unique in a region, hence using timestamp to create unique endpoint name\n",
+    "endpoint_name = \"multimodal-classif-\" + str(timestamp)\n",
```
Comment (rjaincc) [inline sdk/python/foundation-models/system/inference/multimodal-classification/multimodal-classification-batch-endpoint.ipynb:262]:
```suggestion
    "endpoint_name = \"multimodal-multiclass-classif-\" + str(timestamp)\n",
```

File: sdk/python/foundation-models/system/inference/multimodal-classification/multimodal-classification-batch-endpoint.ipynb [hunk 4]
```diff
@@ -0,0 +1,479 @@
...
+    "# Create a batch endpoint\n",
+    "endpoint = BatchEndpoint(\n",
+    "    name=endpoint_name,\n",
+    "    description=\"Batch endpoint for \"\n",
+    "    + foundation_model.name\n",
+    "    + \", for multimodal-classification task\",\n",
```
Comment (rjaincc) [inline sdk/python/foundation-models/system/inference/multimodal-classification/multimodal-classification-batch-endpoint.ipynb:268]:
```suggestion
    "    + \", for multimodal-multiclass-classification task\",\n",
```

File: sdk/python/foundation-models/system/inference/multimodal-classification/multimodal-classification-online-endpoint.ipynb [hunk 5]
```diff
@@ -0,0 +1,409 @@
...
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.8.0"
+  }
+ },
```
Comment (rjaincc) [inline sdk/python/foundation-models/system/inference/multimodal-classification/multimodal-classification-online-endpoint.ipynb:406]: Clean notebook metadata before publishing

**Review by rjaincc — APPROVED** [review]
Approving with minor comments

**Pull request merged**